aten/src/THC/THCScanUtils.cuh

```diff
@@ -4,6 +4,12 @@
 #include "THCAsmUtils.cuh"
 #include "THCDeviceUtils.cuh"
 
+#if defined(__HIP_PLATFORM_HCC__)
+#define WARP_SIZE 64
```

Comment:
Can you give this a more unique name? If someone includes this header somewhere else where `WARP_SIZE` is already defined, you will error.